src/MarketingPageAcceptanceTests.Steps/MarketingPageAcceptanceTests.Steps.csproj

```diff
@@ -21,4 +24,13 @@
     </Compile>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="appsettings.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Update="Utils\tokens.json">
```

Comment:
why do we still need this tokens file, if we now have the app settings?

Reply:
It's been removed in the above commit